apps/fluent-tester/src/TestComponents/Test.tsx

```diff
@@ -63,6 +64,11 @@ const isMobile = Platform.OS == 'android' || (Platform.OS == 'ios' && !Platform.
 
 export const Test = (props: TestProps): React.ReactElement<Record<string, never>> => {
   const [showStatus, setShowStatus] = React.useState(false);
+  const fontBuiltInProps = {
+    fontFamily: 'Arial',
+    codepoint: showStatus ? 0x2796 : 0x2795,
```

Comment:
Maybe leave a comment what these code points are? or do something ike:

```
const plusCodepoint = 0x2796;
const minusCodepoint = 0x2795;
```

Also what happens if we don't specify the font family? Or get the better the family from the theme?

```
const theme = useTheme()
const fontFamily = theme.typography.families.primary

```